src/PerfView/App.cs

```diff
@@ -44,7 +44,7 @@ public static int Main(string[] args)
             try
             {
                 // Can't display on ARM because the SplashScreen is WPF
-                var noGui = SupportFiles.ProcessArch == "ARM" ||
+                var noGui = SupportFiles.ProcessArch == ProcessorArchitecture.Arm ||
```

Comment:
At one point (and it probably is still true), PerfView could collect ETW data on ARM (Mostly by simply avoiding using GUI things since WPF was not supported on ARM).   It has not been used in a while because ARM is just not interesting anymore and even when it was there were signing requirements that got in the way.

However ARM may yet come back for the server.   I would not invest in it right now, but I also would not proactively rip it out (the arm specific things should be small).